external/swe-polybench/scripts/download.py

```
@@ -0,0 +1,131 @@
+        print(f"Failed to load dataset: {e}", file=sys.stderr)
+        return 1
+
+    columns = list(ds.column_names)
+    records = [row_to_record(ds[i], columns) for i in range(len(ds))]
+    print(f"Loaded {len(records)} instances.", file=sys.stderr)
+
+    try:
+        with open(path_500, "w", encoding="utf-8") as f:
+            for r in records:
+                f.write(json.dumps(r, ensure_ascii=False) + "\n")
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)